### PR TITLE
[WIP] Taxonomy generation feature branch

### PR DESCRIPTION
**Pull request by tijmenb**
Trello: https://trello.com/c/9qaQepxu

### CODE REVIEW

File: spec/features/taxonomy_generation/generate_terms_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    when_i_visit_the_taxonomy_project_page
+    and_i_click_to_start_with_a_project
+    then_i_should_see_a_page
+    when_i_submit_a_term
+    then_the_term_is_saved
+    and_i_see_the_next_page
+
+    when_i_go_to_the_project_page
+    then_i_see_the_generated_terms
+  end
```
Comment (tuzz) [inline spec/features/taxonomy_generation/generate_terms_spec.rb]: I really like this style.

Comment (tijmenb) [inline spec/features/taxonomy_generation/generate_terms_spec.rb]: Cool! We started experimenting with it in content-tagger, and it has held up pretty well. 

We got it from FutureLearn: https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec/

**Comment by whoojemaflip** [comment]
Rebased

**Comment by tijmenb** [comment]
Let's not rebase this anymore, it will interfere with any feature branches off this branch. We can merge in master, then when this branch is finished squash & rebase.

**Pull request merged**